packages/vite/src/generators/init/schema.d.ts

```diff
@@ -0,0 +1,3 @@
+export interface Schema {
+  uiFramework: 'react' | 'vue3' | 'vue3-jsx';
```

Comment:
I don't think we care about non-React projects right now, but it's fine leaving this here. Maybe we'll use it later on.